tracer/test/Datadog.Trace.TestHelpers/SpanTagAssertion.cs

```diff
@@ -32,7 +33,14 @@ internal SpanTagAssertion(Result result, IDictionary<string, string> tags)
             .IsOptional(Tags.GitRepositoryUrl)
             .IsOptional(Tags.GitCommitSha);
 
-        public static void AssertNoRemainingTags(SpanTagAssertion s)
+        public static void DefaultMetricAssertions(SpanTagAssertion<T> s) => s
+            .IsOptional("_dd.tracer_kr")
```

Comment:
We're using constants for some (`Tags.GitRepositoryUrl`) and not others. If we are using hard-coded string here for a reason, should we do that for all? Or should we use the constants everywhere?

Reply:
Good question, probably best to use the constants. I just copy pasted @pierotibou's work so I blame him 🤣

Reply:
Good question, probably best to use the constants. I just tried to imitate what @zacharycmontoya  was doing, so I blame him 🤣
(and the ball keeps rolling) :p

Reply:
Actually I changed my mind - they probably _shouldn't_ be constants, given this is meant to be an independent verification of the values 🙂 Removed the Git constants instead

Reply:
Thanks!